deploy/olm-catalog/bundle/manifests/migration.openshift.io_migclusters.yaml

```diff
@@ -47,19 +48,26 @@ spec:
           description: MigClusterSpec defines the desired state of MigCluster
           properties:
             azureResourceGroup:
+              description: AzureResourceGroup is for azure clusters -- it's the resource group that in-cluster volumes use.
               type: string
             caBundle:
+              description: If the migcluster needs SSL verification for connections a user can supply a custom CA bundle.
               format: byte
               type: string
             exposedRegistryPath:
+              description: Stores the path of registry route when using direct migration.
               type: string
             insecure:
+              description: Insecure stores the status of the connection with source cluster. True signifies the connection is insecure.
```

Comment:
This is not a `status`. This is where the user decides if they will validate certificates used in a TLS connection to the remote cluster. If the remote cluster has TLS certs signed by a non standard CA (e.g. self-signed certs), they will need to provide a CABundle to run in secure mode.

Reply:
Updated the description.